client/services/code_preprocessors/PythonCodePreprocessorServiceSpec.js

```diff
@@ -34,6 +34,29 @@ describe('PythonCodePreprocessorService', function() {
       'PerformanceTestObjectFactory');
   }));
 
+  describe('_jsonVariableToPython', function() {
+    it('should correctly convert a json String to a Python string', function() {
+      expect(
+        PythonCodePreprocessorService._jsonVariableToPython('stringify')
+      ).toEqual("'stringify'");
+    });
+
+    it('should correctly convert a json Array to a Python array (or a string version)'
+      , function() {
+      expect(
+        PythonCodePreprocessorService._jsonVariableToPython(["cat", "2", "3"])
+      ).toEqual("['cat', '2', '3']");
+    });
+
+    it('should correctly convert a nested json Array to a similar Python array)'
+      , function() {
+      expect(
+        PythonCodePreprocessorService._jsonVariableToPython(
+          [["1", "2"], ["3", "4"], ["5", "6"]])
+      ).toEqual("[['1', '2'], ['3', '4'], ['5', '6']]");
+    });
+  });
```

Comment:
So we actually don't support booleans as input (or didn't as of your comment), but we do now.